### ISSUE TITLE
VorbisWaveReader throws "Ran out of packets?!" exception when setting CurrentTime

### ISSUE THREAD

**Issue by PKBeam** [naudio/Vorbis#14]
I have a Vorbis file ([test.zip](https://github.com/naudio/Vorbis/files/7149680/test.zip)) that causes `VorbisWaveReader` to throw "Ran out of packets?!" exceptions when setting the `CurrentTime` to certain values.

Here's the code I used to test:
```c#
VorbisWaveReader vwr = new("test.ogg");
List<int> failures = new();
for (int i = 0; i <= (int)vwr.TotalTime.TotalMilliseconds; i++) {
    try {
        vwr.CurrentTime = TimeSpan.FromMilliseconds(i);
    } catch {
        failures.Add(i);
    }
}
```
The `failures` array ends up with 1137 values ranging from 2724 to 201600.
The array remains the same after I re-run the code.

The file was made with Audacity and ffmpeg doesn't complain when handed the file, so I don't think it's corrupt or invalid.

**Comment by ioctlLR** [comment]
Yeah it's not your file.  There's a subtle issue in NVorbis that I haven't figured out yet.  https://github.com/NVorbis/NVorbis/issues/19 tracks it.

**Comment by ioctlLR** [comment]
Found it (with some help).  https://github.com/NVorbis/NVorbis/pull/25 will fix it in NVorbis; I'll release v0.10.4 shortly after merging and publish a new release here that targets it immediately after.

**Comment by ioctlLR** [comment]
Closed by v1.5.0 release
